app/routes/users/components/Penalties.css

```diff
@@ -1,3 +1,41 @@
-.penalty {
-  margin-bottom: 10px;
+.weight {
+  color: var(--danger-color);
+  font-size: var(--font-size);
+  font-weight: 500;
+}
+
+.eventLink {
+  color: var(--secondary-font-color);
+  transition: color var(--easing-fast);
+
+  &:hover {
+    color: var(--color-gray-6);
+  }
+}
+
+.info {
+  line-height: 1.3;
+
+  & :first-child {
```

Comment:
```suggestion
  &:first-child {
```

Reply:
Yea no. For reasons beyond science this does not work